docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,225 @@
+) and click **NEXT**.
+
+3. Teleport Connect will ask you for your username, password, and MFA. If this
+Teleport plane is integrated with an external Identity Provider (**IdP**), it
+may instead prompt you to authenticate in a browser window with that service.
+
+4. Once authenticated, you can browse and connect to all the resources your
+user is permitted to access:
+
+   ![An example of Teleport Connect, populated with instances](../img/use-teleport/connect-ui-overview.png)
+
+## Server Access
+
+<Tabs>
+<TabItem label="tsh">
+
+`tsh ls` lists the servers you have access to through Server Access (ssh):
+
+```code
+$ tsh ls
+Node Name           Address         Labels                       
+------------------- --------------- ---------------------------- 
+server1.example.com 192.0.2.24:3022 access=servers,hostname=server1
+server2.example.com 192.0.2.32:3022 access=servers,hostname=server2
+```
+
+To connect to a server:
+
+```code
+$tsh ssh server1.example.com
+root@server1:~$
+```
+
+</TabItem>
+<TabItem label="Teleport Connect">
+
+Under the **Servers** tab, click **CONNECT** next to the server you want to
+access. Select or type in a username available to your Teleport user.
+
+In a new tab, Teleport Connect will initiate the connection and provide a terminal environment.
+
+</TabItem>
+</Tabs>
+
+## Transferring files
+
+`tsh scp` will let you transfer files to servers behind Teleport:
+
+```code
+$ tsh scp somefile.ext xeon.tp.alexf.xyz:
+somefile.ext   7% |███████                | (25/342 MB, 2.9 MB/s) [9s:1m48s]
+```
+
+## Database Access
+
+<Tabs>
+<TabItem label="tsh">
+
+`tsh db ls` will list the databases available to your user:
+
+```code
+$ tsh db ls
+Name                                               Description Allowed Users   Labels  Connect
+-----------------------------------------------    ----------- --------------- ------- -------------------------
+myelastic                                                      [alice elastic] env=dev
+mysql-server1 (user: alice, db: teleport_example)              [alice elastic] env=dev tsh db connect mysql-server1
+```
+
+To connect to a database server, through `tsh`, you'll need a local client for
+that database. For example, to connect to a MySQL or MariaDB database, you'll
+need the [MySQL CLI](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) client:
+
+```code
+tsh db connect --db-user=alice --db-name=teleport_example mysql-server1 
+```
+
+In this example, `teleport_example` is a pre-existing database on the MySQL server:
+
+```sql
+SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| mysql              |
+| performance_schema |
+| teleport_example   |
++--------------------+
+4 rows in set (0.09 sec)
+```
+
+For other connection types, you can create a local tunnel to point your software to:
+
+```code
+$ tsh proxy db myelastic --db-user=alice --tunnel
```

Comment:
DB tunneling is not very popular unless it's being used with Teleport Connect or another DB client. If we want to keep these docs concise this section could be removed.

Reply:
I updated the preceding sentence with what I think makes this section relevant for end-users; access for graphic database clients. I'm no DB admin so I can't say how popular they are versus CLI tools, but my experience creating the Elasticsearch Database Access guide required using a tunnel since using the elastic CLI tool on a system (especially a Mac) without having full Elasticsearch installed locally was a 🤬 and a half.

Keeping in mind that this page is supposed to cover 90% of end users (and not necessarily DB-admins/dev/eng/etc folks, the question is: do we expect the target audience of this page to use GUI database clients?